### PR TITLE
[RFR]: renamed St2 and other terms in docs

### PR DESCRIPTION
**Pull request by yhasan**
Made the following changes:
1) Stackstorm -> to mostly |st2|, some Brocade Workflow Composer, some Stackstorm, 5+ -> Brocade (submit_debug_info.rst) 
2) Flow -> Workflow Designer (enterprise.rst)
3) Stackstorm community -> StackStorm
4) Enterprise Edition -> Brocade Workflow Composer 

Created PR, so folks can review. Will resolve TOC conflict with Dmitri later

### CODE REVIEW

File: docs/source/_themes/sphinx_rtd_theme/layout.html [hunk 1]
```diff
@@ -17,10 +17,11 @@
   <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
   {% endblock %}
 
-  <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}?v4"/>
+  <link rel="shortcut icon" href="{{ pathto('_static/favicon.ico', 1) }}?brocade"/>
```
Comment (dzimine) [inline docs/source/_themes/sphinx_rtd_theme/layout.html]: why is this here? afaik the whole _themes for brocade is coming from ipfabric-docs, https://github.com/StackStorm/ipfabric-docs/tree/master/docs/source/_themes/sphinx_rtd_theme

Comment (dzimine) [inline docs/source/_themes/sphinx_rtd_theme/layout.html]: Is this a "mistakenly added file"?

File: docs/source/_themes/sphinx_rtd_theme/static/css/theme.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿@charset "UTF-8";
```
Comment (dzimine) [inline docs/source/_themes/sphinx_rtd_theme/static/css/theme.css]: Don't you commit this from ipfabric by mistake? I afraid this will paint st2 docs brocade colors :)

File: docs/source/chatops/aliases.rst [hunk 3]
```diff
@@ -166,7 +166,7 @@ A single alias file allow multiple formats to be specified for a single alias e.
     ---
     name: "st2_sensors_list"
     action_ref: "st2.sensors.list"
-    description: "List available StackStorm sensors."
```
Comment (dzimine) [inline docs/source/chatops/aliases.rst]: wrong: this won't work as |st2| won't expand in code blocks. 
Just keep it as StackStorm.

File: docs/source/chatops/chatops.rst [hunk 4]
```diff
@@ -13,7 +13,7 @@ be done via ChatOps.
 
 Not only does ChatOps reduce the feedback loop of work output, it also
 empowers others to accomplish complex self-service tasks that they
-otherwise would not be able to do. Combining ChatOps and StackStorm is
+otherwise would not be able to do. Combining ChatOps and |st2| is
```
Comment (dzimine) [inline docs/source/chatops/chatops.rst]: Just a suggestion: given the appeal of Chatops to opensource, keep it as StackStorm here

File: docs/source/_themes/sphinx_rtd_theme/static/css/theme.css (hunk 2 above)
Comment (LindsayHill) [inline docs/source/_themes/sphinx_rtd_theme/static/css/theme.css]: I'm in the process of fixing this stuff. @yhasan's original commit included a bunch of brocade/ipfabric stuff, and I'm in the process of unwinding it. Should be fixed in latest commit

File: docs/source/_themes/sphinx_rtd_theme/layout.html (hunk 1 above)
Comment (LindsayHill) [inline docs/source/_themes/sphinx_rtd_theme/layout.html]: Yep

File: docs/source/chatops/index.rst [hunk 5]
```diff
@@ -24,7 +24,7 @@ Interested in learning more? Here are some things to get you started on your voy
 
 * `ChatOps: Technology and Philosophy <https://www.youtube.com/watch?v=IhzxnY7FIvg>`_
 * `Start automating with Slack <https://medium.com/why-not/what-will-the-automated-workplace-look-like-495f9d1e87da>`_
-* `Demonstration: See StackStorm and ChatOps in action <https://www.youtube.com/watch?v=fUpSaEOS_BA>`_
+* `Demonstration: See |st2| and ChatOps in action <https://www.youtube.com/watch?v=fUpSaEOS_BA>`_
```
Comment (dzimine) [inline docs/source/chatops/index.rst]: Same as above - keep at StackStorm with Chatops.

File: docs/source/development/index.rst [hunk 6]
```diff
@@ -1,7 +1,7 @@
 Development
 ===========
 
-This page describes StackStorm development process and contains general
+This page describes |st2| development process and contains general
```
Comment (dzimine) [inline docs/source/development/index.rst]: We DON"T opensource BWC thus here in 'contribution/development' we better say "StackStorm".

File: docs/source/development/testing.rst [hunk 7]
```diff
@@ -1,7 +1,7 @@
 Testing
 =======
 
-In StackStorm we have a multiple types of tests described bellow.
+In |st2| we have a multiple types of tests described bellow.
```
Comment (dzimine) [inline docs/source/development/testing.rst]: same: for development section, keep StackStorm.

File: docs/source/install/rhel6.rst [hunk 8]
```diff
@@ -386,19 +386,19 @@ If you already run Hubot instance, you only have to install the `hubot-stackstor
...
-Upgrade to Enterprise Edition
------------------------------
-Enterprise Edition is deployed as an addition on top of StackStorm Community. You will need an active
-Enterprise subscription, and a license key to access StackStorm enterprise repositories. To add your
-license key, replace ``${ENTERPRISE_LICENSE_KEY}`` in the command below with the key you received when
+Upgrade to Brocade Workflow Composer
+-------------------------------------
+Brocade Workflow Composer is deployed as an addition on top of StackStorm. You will need an active
+Brocade Workflow Composer subscription, and a license key to access Brocade Workflow Composer repositories.
+To add your license key, replace ``${BWC_LICENSE_KEY}`` in the command below with the key you received when
```
Comment (dzimine) [inline docs/source/install/rhel6.rst]: We may need to update ${ENTERPRISE_LICENSE_KEY} everywhere? Hope you cover it here, but there may be scripts, emails, so let's sniff for changes outside of docs.

Comment (LindsayHill) [inline docs/source/install/rhel6.rst]: Ugh. Yeah. Could be some emails that refer to it. Don't think it's a problem with scripts any more since we don't do AIO.

File: docs/source/rbac.rst [hunk 9]
```diff
@@ -3,8 +3,8 @@ Role Based Access Control
 
 .. note::
 
-   Role Based Access Control (RBAC) is only available in StackStorm Enterprise Edition. For
-   information about enterprise edition and differences between community and enterprise edition,
+   Role Based Access Control (RBAC) is only available in Brocade Workflow Composer. For
+   information about Brocade Workflow Composer and differences between Stackstorm and Brocade Workflow Composer,
```
Comment (LindsayHill) [inline docs/source/rbac.rst]: Yes. It's a change I'd like to make through the docs.

**Comment by dzimine** [comment]
ok to merge, before it goes crazy again :) but please review the comments inline and fix some of them.
